NMS-13472: Update table formatting in docs (continued) (#3564)
OpenNMS/opennms: docs/modules/operation/pages/elasticsearch/features/event-forwarder-mapping.adoc

Thus, each index entry contains more context information than would be found in the actual {page-component-title} event.
This context information includes the associated node and asset table information which was current when (but may have changed since) the event was archived.

> The <<elasticsearch/features/event-forwarder-mapping-table.adoc#ga-elastic-field-index, table of index mappings>>  has an example event JSON entries retrieved using a sense command.

Author: ```suggestion
The <<elasticsearch/features/event-forwarder-mapping-table.adoc#ga-elastic-field-index, table of index mappings>> illustrates sample event data as stored in Elasticsearch.
```
